roles/cloud_connector/templates/foreman-rh-cloud-worker.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+CONFIG_FILE="{{ cloud_connector_config_file }}" /usr/libexec/yggdrasil-worker-forwarder
```

Comment:
Since this is just a wrapper to set an env variable, it should probably exec into the the worker to not leave shells running around
```suggestion
CONFIG_FILE="{{ cloud_connector_config_file }}" exec /usr/libexec/yggdrasil-worker-forwarder
```